src/utils/openmc/universe.py

```diff
@@ -1079,18 +1157,22 @@ def __repr__(self):
             string += '{0: <16}{1}{2}\n'.format('\tOuter', '=\t',
                                                 self._outer)
 
-        string += '{0: <16}\n'.format('\tUniverses')
+        string += '{0: <16}'.format('\tUniverses')
 
-        # FIXME: This loop must be revised for hexagonal lattice ordering
-        # Lattice nested Universe IDs - column major for Fortran
-        for i, universe in enumerate(np.ravel(self._universes)):
-            string += '{0} '.format(universe._id)
+        z = 0
+        while True:
```

Comment:
Ditto my earlier comment - I don't see any reason why this code block should be in a ``while True`` code block. Unless I'm missing something, break it up into a sequence of ``if-else`` statements instead.